scripts/covid19_india/medical_tests_in_data/manifest.json

```diff
@@ -0,0 +1,18 @@
+{
+    "import_specifications": [{
+        "import_name":
+        "medical_tests_in_data",
+        "curator_emails": ["hhhhhwu@google.com"],
```

Comment:
Update this?

Reply:
Can I add my personal email or does this need to be a google id?

Reply:
Non-google.com ID works. Though if you prefer to not have your email ID here, we can certainly find a placeholder ID.